apps/dashboard/src/api/e2e-sdk.helper.ts

```diff
@@ -0,0 +1,56 @@
+import { SDKOptions } from '@novu/api/lib/config';
+// This is how we import the speakeasy autogenerated Novu SDK that is CJS in a the Dashboard ESM project with Vite
+// Read more at https://github.com/vitejs/vite/issues/5668#issuecomment-968117934
+/** DO NOT CHANGE THIS CODE START */
+import * as NovuAPI from '@novu/api';
+import { IEnvironment } from '@novu/shared';
+import { getToken } from '@/utils/auth.ts';
+import { API_HOSTNAME } from '@/config';
+import * as NovuHttpClient from '@novu/api/lib/http';
+
+const { Novu } = NovuAPI;
+const { HTTPClient } = NovuHttpClient;
+
+/** DO NOT CHANGE THIS CODE END */
```

Comment:
is it possible to minimize the do not touch block, all the imports are relevant to the comment?? 

```suggestion
// Read more at https://github.com/vitejs/vite/issues/5668#issuecomment-968117934
/** start - DO NOT CHANGE THIS CODE */
import * as NovuAPI from '@novu/api';
/** end - DO NOT CHANGE THIS CODE */
```